libcst/codegen/gen_visitor_functions.py

```diff
@@ -0,0 +1,163 @@
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pyre-strict
+import inspect
+from collections import defaultdict
+from typing import Dict, Generator, List, Mapping, Sequence, Set, Type
+
+import libcst as cst
+
+
+DO_NOT_CARE_TYPES: Set[str] = {"CSTNode", "ABC", "object"}
+
+
+def _get_nodes() -> Generator[Type[cst.CSTNode], None, None]:
+    """
+    Grab all CSTNodes that are not a superclass. Basically, anything that a
+    person might use to generate a tree.
+    """
+
+    # pyre-ignore __dir__ exists on all objects.
+    for name in cst.__dir__():
+        if name.startswith("__") and name.endswith("__"):
+            continue
+        if name == "CSTNode":
+            continue
+
+        node = getattr(cst, name)
+        try:
+            if issubclass(node, cst.CSTNode):
```

Comment:
As-is, this seems to overlap a bit with your `DO_NOT_CARE_TYPES` check later. This is going to filter out `ABC` and `object`.